snap/gadget_test.go

```diff
@@ -119,6 +120,36 @@ defaults:
       bar: baz
 `)
 
+var mockVolumeUpdateGadgetYaml = []byte(`
+volumes:
+  bootloader:
+    schema: mbr
+    bootloader: u-boot
+    id:     id,guid
+    structure:
+      - filesystem-label: system-boot
+        offset: 12345
```

Comment:
Hm, I know this is just a mock but I don't think its possible to have "offset" and "filesystem" in the same structure item?

Reply:
If I'm reading ubuntu-code correctly, using offset here is actually possible. What's not, is using image `content` when a filesystem is specified. I've updated the mock yaml accordingly.